questionnaire/utils/isQuestionnaireVersionCompatible/index.test.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+beforeEach(() => {
+    jest.clearAllMocks();
+    jest.resetModules();
+});
+
+jest.doMock('semver/functions/diff', () => version => {
```

Comment:
Don't mock things we don't own. Mock the template instance instead. Instantiate a template with a specific version in the json, and compare that against a specific question template version, again, that can be mocked (via the getInstalledModuleVersion function)